apps/statistikk-api/README.md

```diff
@@ -0,0 +1,9 @@
+## testnorge-statistikk-api
+API for statistikk.
+
+### Swagger
+Swagger finnes under [/api](https://testnorge-statistikk-api.nais.preprod.local) -endepunktet til applikasjonen.
+
+### Lokal kjøring
+Kjør StatistikkApiApplicationStarter med følgende argumenter:
+ - -Dspring.profiles.active=dev
```

Comment:
Jeg måtte legge til vault token også for å få det til å kjøre. Skulle det også stått her eller gjør jeg noe merkelig?

Reply:
Nei du rett :)